.github/workflows/ci.yaml

```diff
@@ -102,7 +102,7 @@ jobs:
       - name: Set up Node
         uses: actions/setup-node@v2
         with:
-          node-version: '16'
+          node-version: '18'
```

Comment:
Should we specify the minor version here as well?

Reply:
Yeah, we could. However, we would also need to remember to update it over time as new LTS minor releases came out. The good thing as of now is since 18 is currently in LTS, each minor beyond that should also be LTS. I don't think if anyone tries to get an 18 from today onwards we should ever go backwards to anything before LTS, either.